clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -51,9 +51,13 @@
     private static final String PARTITION_HEADER_KEY_NAME = "partition_header";
     private static final String PARTITION_KEY_NAME = "partition";
     private static final String ERROR_CODE_KEY_NAME = "error_code";
+    private static final String HIGH_WATERMARK_KEY_NAME = "high_watermark";
+    private static final String LOG_START_OFFSET_KEY_NAME = "log_start_offset";
 
     // Default throttle time
     private static final int DEFAULT_THROTTLE_TIME = 0;
+    // Default low watermark
+    private static final long DEFAULT_LOG_START_OFFSET = 0L;
```

Comment:
I don't think it is necessary to distinguish between NO_LOG_START_OFFSET v.s. LOG_START_OFFSET = 0. Is there any use-case for NO_LOG_START_OFFSET?